wagtail/admin/viewsets/model.py

```diff
@@ -195,6 +199,7 @@ def get_urlpatterns(self):
     def on_register(self):
         super().on_register()
         self.register_admin_url_finder()
+        self.register_reference_index()
```

Comment:
Worth noting that this will enable reference tracking for the handful of models that have a ModelViewSet within Wagtail as standard - that would be Sites, Collections, Locales, Users and Groups I think. I don't think there are any negative consequences to that, though - after all, we were indexing those (along with all other models project-wide) in Wagtail 4.1 - 4.2.

Reply:
Good catch, thanks! I think it's worth adding an `add_to_reference_index` flag to the `ModelViewSet`. I also think we should turn indexing off for those models at least for now, in case that would slow down instances that have many foreign keys to those models. The models specifically are Site, Locale, and Group, as the Collection and User views haven't been migrated to use `ModelViewSet`.

I've done so and added upgrade considerations & release notes.